src/sc2utils/sc2_manage_process.cc

```diff
@@ -238,7 +238,7 @@ bool TerminateProcess(uint64_t process_id) {
     if (index < 0)
         return false;
 
-    ::TerminateProcess(windows_processes[index].pi_.hProcess, -1);
+    ::TerminateProcess(windows_processes[index].pi_.hProcess, static_cast<UINT>(-1));
```

Comment:
Ah, I didn't see that macro and also assumed it was calling the definition in the file. Thanks for the clarification :)